### PR TITLE
Fix SnowMaskTask when dealing with empty patches

### PR DESCRIPTION
**Pull request by mlubej**
Was easier to return something early, instead of fixing the shapes

### CODE REVIEW

**Comment by zigaLuksic** [comment]
cv2 is really making our life difficult with all its weird custom types :upside_down_face:

**Review by zigaLuksic — COMMENTED** [review]
where did this fail before? I dont see any explicit loops in the execute body

would be great to have a test :D

**Comment by mlubej** [comment]
> where did this fail before? I dont see any explicit loops in the execute body

It happens in the `_apply_dilation` method

Will update

File: tests/mask/test_snow_mask.py [hunk 1]
```diff
@@ -33,3 +33,12 @@ def test_snow_coverage(task, result, test_eopatch):
...
     assert snow_pixels[-4] == result[1], "Snowy pixels on specified frame do not match"
+
+
+def test_snow_empty_eopatch(test_eopatch):
+    bands = test_eopatch.data["BANDS-S2-L1C"]
+    empty_bands_array = np.array([], dtype=bands.dtype).reshape((0, *bands.shape[1:]))
+    empty_eopatch = EOPatch(bbox=test_eopatch.bbox, timestamps=[], data={"BANDS-S2-L1C": empty_bands_array})
+    task = SnowMaskTask((FeatureType.DATA, "BANDS-S2-L1C"), [2, 3, 7, 11], mask_name="TEST_SNOW_MASK")
+    resulting_eopatch = task(empty_eopatch) # checks if the task runs without errors
+    assert resulting_eopatch.mask["TEST_SNOW_MASK"].shape[0] == 0
```
Comment (zigaLuksic) [inline tests/mask/test_snow_mask.py:44]: i would test on the entire shape

**Pull request merged**